axelor-human-resource/src/main/resources/views/TimesheetLine.xml

```diff
@@ -149,6 +165,36 @@
 			<field name="user"  form-view="user-form" grid-view="user-grid" hidden="true" canNew="false" canEdit="false" canView="false"/>
 		</panel>
 	</form>
+	
+	<form model="com.axelor.apps.hr.db.TimesheetLine" title="Timesheet Line" name="timesheet-line-timesheet-teamtask-form" onNew="action-team-task-attrs-default-timesheet-line,action-team-task-group-timesheet-line-set-timesheet" onSave="action-team-task-group-timesheet-line-onsave">
+		<panel name="overviewPanel" title="Overview">
+			<field name="project" onChange="action-timesheet-line-group-project-onchange" onSelect="action-timesheet-line-attrs-domain-project" form-view="project-form" grid-view="project-grid"/>
+			<field name="teamTask" form-view="team-task-form" grid-view="team-task-grid" onSelect="action-timesheet-line-attrs-domain-team-task" onChange="action-timesheet-line-method-set-default-to-invoice"/>
```

Comment:
what is the use of onChange and onSelect actions on project and teamTask?
these fields are readOnly